{{cookiecutter.app_name}}/requirements/dev.txt

```diff
@@ -16,3 +16,4 @@ flake8-docstrings==1.5.0
 flake8-isort==2.7.0
 isort==4.3.21
 pep8-naming==0.9.1
+pipenv==2018.11.26
```

Comment:
If a user chooses to use `pipenv` in their project, the [post gen hook](https://github.com/cookiecutter-flask/cookiecutter-flask/blob/1e34130ad536b3159ec3e70e75a8f9687f5f49a1/hooks/post_gen_project.py#L20) will delete `requirements.txt` and `requirements/*` from the generated project. As a result, I don't believe adding this requirement here will have the desired effect.